### PR TITLE
🎨( feat ) Delete block#14

### PR DESCRIPTION
**Pull request by CarolGitonga**
When a user clicks delete on the side menu of a block, the block needs to be deleted from the editor. Upon saving and reloading, the block needs to be removed

Fixes # (# 14) 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/block/block.component.html [hunk 1]
```diff
@@ -17,15 +17,36 @@ <h4 class="block-title">{{block.blockTitle}}</h4>
...
+
+    <div *ngSwitchCase="questiontype">
+        <app-questions-block [id]="id" [block]="block" [questionMessageBlock]="blockFormGroup" [jsPlumb]="jsPlumb" class="block"></app-questions-block>
+    </div>
+
+    <div *ngSwitchCase="locationtype">
+        <app-location-block [id]="id" [block]="block" [locationMessageForm]="blockFormGroup" [jsPlumb]="jsPlumb" class="block"></app-location-block>
+    </div>
+
+>>>>>>> main
```
Comment (pmutua) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/block/block.component.html:50]: @CarolGitonga you missed out on removing `>>>>>>> main` when you were resolving the conflicts. Remove this.

**Review by pmutua — CHANGES_REQUESTED** [review]

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/block/block.component.html (hunk 1 above)
Comment (CarolGitonga) [reply libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/block/block.component.html:50]: Okay

**Comment by CarolGitonga** [comment]
@pmutua .I have removed the code

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/block/block.component.ts [hunk 2]
```diff
@@ -28,6 +28,15 @@ export class BlockComponent implements OnInit {
   @Input() block: StoryBlock;
   @Input() blocksGroup: FormArray;
   @Input() jsPlumb: BrowserJsPlumbInstance;
+  
+  @Output() deleteBlock: EventEmitter<string> = new EventEmitter<string>();
+  
+  delete()  {
+    let confirmDelete = confirm('Are you sure you want to delete this block')
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/block/block.component.ts:35]: If there is a confirm dialog, would prefer to have a separate dialog component for this. So just remove the confirm for now

**Review by ReaganCn — CHANGES_REQUESTED** [review]
The block is not being deleted, based on your implementation. As the block needs to be removed upon saving/reloading.

You can record a short video to demonstrate this if it is working.

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/block/block.component.ts (hunk 2 above)
Comment (CarolGitonga) [reply libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/block/block.component.ts:35]: @ReaganCn Okay

**Comment by jrosseel** [comment]
This implementation is missing a state interaction and thus not removing the block

**Pull request closed without merging**